### PR TITLE
Fix writing of image into file

### PR DESCRIPTION
**Pull request by rshym**
Due to mistake total size of image was calculated after it's usage.
Now DEFAULT_IMAGE_SIZE_GIB is calculated before inflate_image().

Signed-off-by: Ruslan Shymkevych <ruslan_shymkevych@epam.com>

### CODE REVIEW

File: doc/mk_sdcard_image.sh [hunk 1]
```diff
@@ -3,12 +3,12 @@
...
 CUR_STEP=1
 DOMA_PART_N=p3
-# see partition_image() for definition of sizes of partitions
+# sizes of partitions are defined in set of variables DOM*_START, DOM*_END
 
 usage()
 {
 	echo "###############################################################################"
-	echo "SD card image builder script v1.0"
+	echo "SD card image builder script v1.1"
```
Comment (andr2000) [inline doc/mk_sdcard_image.sh]: this could be a dedicated commit if your will

File: doc/mk_sdcard_image.sh [hunk 2]
```diff
@@ -465,18 +433,51 @@ if [ -z "$doma_name" ]; then
 	exit 2
 fi
 
+# Define partitions for different products.
```
Comment (andr2000) [inline doc/mk_sdcard_image.sh]: Can we move this piece to a dedicated function on top, so it has its own space not being part of other func?
This will allow us using it w/o any further move and improve readability, e.g. once you open the script you see the image partitioning

**Comment by rshym** [comment]
Comments are addressed.
Tested:
* write to file
* write to card
* update doma on card

**Comment by andr2000** [comment]
`Reviewed-by: Oleksandr Andrushchenko <oleksandr_andrushchenko@epam.com>`
